### DISCUSSION TITLE
[Help Wanted] How to edit a Smart View Query once it is created?

### DISCUSSION THREAD

**Discussion by patgadget** [standardnotes/forum#2549] [General] [answered]
Is there a way that we could edit the Smart View query once created.
It seams that the only thing that we can edit is the Name of the search, but not the query itself.

Any idea?
Thanks
Pat Gadget

**Comment by myreli** [comment, marked as answer]
Hi @patgadget! Sorry for the late reply. I'm triaging older issues and just saw your question.  

Detailed instructions can be found here: https://standardnotes.com/help/42/how-do-i-view-a-list-of-untagged-notes-and-create-other-dynamic-filters

We now have an edit panel for existing views as well as a simple builder for new Smart Views:
<img width="1012" alt="image" src="https://user-images.githubusercontent.com/17554234/218784855-1f50b043-fdeb-4a73-8c40-004b3dbca901.png">
<img width="1012" alt="image" src="https://user-images.githubusercontent.com/17554234/218785276-469b1843-2208-43b7-8c06-9d371aa84539.png">
